tests/Unit/Framework/ActionTesting.hpp

```diff
@@ -471,8 +471,10 @@ using item_type_if_contained_t =
 // `emplace_component_and_initialize` function.
 template <typename... SimpleTags, typename ComputeTagsList>
 struct InitializeDataBox<tmpl::list<SimpleTags...>, ComputeTagsList> {
-  using simple_tags = tmpl::list<SimpleTags...>;
-  using compute_tags = ComputeTagsList;
+  // these type aliases must be different from those used by `SetupDataBox` to
+  // avoid a clash.
```

Comment:
I would just like to note here that a protocol would resolve this clash :) The `SetupDataBox` action would only pick up those actions that indicate they conform to the protocol, making the coupling explicit.

Not requesting a change here, I agree with introducing protocols on smaller scales first, just wanted to point this out.